[UIK-3844][input-tags] preserve typed text when pasting (#2441)
semrush/intergalactic: semcore/input-tags/CHANGELOG.md

CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).

## [16.0.9] - 2025-09-12

### Fixed

> - Entered text is pasted when a new text is pasted afterwards.

Reviewer: ```suggestion
- Typed text is lost when new text is pasted after it.
```
